compose.yaml

```diff
@@ -30,6 +30,18 @@ services:
       - "traefik.http.services.backend.loadbalancer.server.port=80"
     networks:
       - rope
+  postgres:
+    image: postgres:15
```

Comment:
I'm thinking we'll use version 16 in the eventual RDS deployment. Let's go ahead and use that here too so the versions align.